app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_create :update_likes_counter
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [OPTIONAL] As you use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_create` to call the `#update_likes_counter` method every time a new like has been created to the database, then make sure the `update_likes_counter` method is private 🙃 🙃